src/app/Vitessce.integration.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-len */
```

Comment:
This test can be uncommented but I simply could not get the `setTimeout` to actually work on Travis.